pyroomacoustics/room.py

```diff
@@ -1743,7 +1744,7 @@ class ShoeBox(Room):
 
     def __init__(self,
                  p,
-                 fs=8000,
+                 fs=16000,
```

Comment:
I don't think we should change the defaults. This will break code downstream.

Reply:
agreed